fairseq/training.py

```diff
@@ -154,7 +153,7 @@ def __init__(
         :param tk.Path fairseq_python_exe: File path to the executable for running python
         :param tk.Path fairseq_root: File path to the fairseq git for alternative call of fairseq-hydra-train
             (no need to install fairseq here)
-        :param bool use_cache_manager: enables caching of data given in the manifest with the i6 cache manager
+        :param str cache_manager: enables caching of data given in the manifest with cache manager
```

Comment:
I would explicitly list the available options
```suggestion
        :param str cache_manager: if given, enables caching of data given in the manifest with cache manager
            possible values: None: no caching, "i6": use the cache manager, "general": apply gs.file_caching
```

maybe also add an assert that only these three options are given.